symbolic-minidump/benches/from_minidump.rs

```diff
@@ -0,0 +1,40 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use symbolic_common::ByteView;
+use symbolic_minidump::cfi::CfiCache;
+use symbolic_minidump::processor::{FrameInfoMap, ProcessState};
+use symbolic_testutils::fixture;
+
+pub fn minidump_benchmark(c: &mut Criterion) {
+    let buffer = ByteView::open(fixture("linux/mini.dmp")).unwrap();
+    let mut frame_info = FrameInfoMap::new();
+
+    let cfi_records = {
+        let file = BufReader::new(File::open(fixture("linux/crash.sym")).unwrap());
+
+        // Read STACK CFI records starting at line 170
+        file.lines()
+            .skip(169)
```

Comment:
curious why?

Reply:
What I mean here is that line 170 is where the first STACK CFI record is. I should probably clarify the comment.